torch/export/exported_program.py

```diff
@@ -681,6 +681,28 @@ def _decompose_exported_program(
     return exported_program
 
 
+@contextmanager
+def _disabled_load_state_dict_hooks(mod: torch.nn.Module):
+    state_dict_hooks: Dict[int, Callable] = dict(mod._state_dict_hooks)
+    state_dict_pre_hooks: Dict[int, Callable] = dict(mod._state_dict_pre_hooks)
+    mod._state_dict_hooks.clear()
+    mod._state_dict_pre_hooks.clear()
+    try:
+        yield
+    finally:
+        mod._state_dict_hooks = state_dict_hooks
+        mod._state_dict_pre_hooks = state_dict_pre_hooks
+
+
+def exported_program_state_dict(mod: torch.nn.Module):
```

Comment:
let's keep this private for now, and not in this file... maybe in utils?

Reply:
> let's keep this private for now, and not in this file... maybe in utils?

moved to utils now.